assets/src/apps/delivery/layouts/deck/components/FeedbackRenderer.tsx

```diff
@@ -30,19 +24,36 @@ const FeedbackRenderer: React.FC<FeedbackRendererProps> = ({ feedbacks, snapshot
     [snapshot],
   );
 
-  // TODO: other handlers for parts, "advanced" things like tracking part responses within feedback??
-
   return (
-    <div className="feedback-item">
+    <Fragment>
```

Comment:
🎨 React auto-includes (no import required) a shorthand version of `<Fragment>` as: `<>` and `</>`.

Reply:
Forgot to mention, the only caveat is that the shorthand version doesn't support the `key` prop if you need it...